src/components/templates/Camp/components/Map.tsx

```diff
@@ -109,495 +90,156 @@ export const Map: React.FC<MapProps> = ({
-                  return (
-                    <div key={`row${rowIndex}`} className={`seat-row`}>
-                      {venue.showGrid && (
-                        <div
-                          className={"seat-container"}
-                          onClick={() =>
-                            onSeatClick(row, column, seatedPartygoer)
-                          }
-                        >
-                          <div
-                            className={seatedPartygoer ? "seat" : `not-seat`}
-                            key={`row${rowIndex}`}
-                          >
-                            {seatedPartygoer && (
-                              <div className={isMe ? "user avatar" : "user"} />
-                            )}
-                          </div>
-                        </div>
-                      )}
-                    </div>
-                  );
-                })}
-                {venue.showGrid &&
-                  rows &&
-                  partygoers.map((partygoer, index) => {
-                    const isMe = partygoer.id === user?.uid;
-                    const position = partygoer?.data?.[venue.id];
-                    const currentRow = position?.row ?? 0;
-                    const currentCol = position?.column ?? 0;
-                    const avatarWidth = 100 / columns;
-                    const avatarHeight = 100 / rows;
-                    return (
-                      !!partygoer.id && (
-                        <UserProfilePicture
-                          key={`partygoer-${index}`}
-                          user={partygoer}
-                          containerStyle={{
-                            display: "flex",
-                            width: `${avatarWidth}%`,
-                            height: `${avatarHeight}%`,
-                            position: "absolute",
-                            cursor: "pointer",
-                            transition:
-                              "all 1400ms cubic-bezier(0.23, 1 ,0.32, 1)",
-                            top: `${avatarHeight * (currentRow - 1)}%`,
-                            left: `${avatarWidth * (currentCol - 1)}%`,
-                            justifyContent: "center",
-                          }}
-                          avatarStyle={{
-                            width: "80%",
-                            height: "80%",
-                            borderRadius: "100%",
-                            alignSelf: "center",
-                            backgroundImage: `url(${partygoer?.pictureUrl})`,
-                            backgroundSize: "cover",
-                          }}
-                          avatarClassName={`${
-                            isMe ? "me profile-avatar" : "profile-avatar"
-                          }`}
-                          setSelectedUserProfile={setSelectedUserProfile}
-                          miniAvatars={venue?.miniAvatars}
```

Comment:
this was refactored into `src/components/templates/Camp/components/MapPartygoersOverlay.tsx`